db-migrations/src/main/resources/db/migration/V56__deltaker_add_kilde.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE deltaker
```

Comment:
Må den være nullable? Vi kan jo bare sette `DEFAULT 'ARENA'` for nå, selv om det er noen få som burde være `KOMET` i dev så går vel det fint.

Reply:
Kan sette default ARENA :)